docs/api-reference/json.rst

```diff
@@ -158,11 +158,16 @@ Project
                     "yanked": false,
                     "yanked_reason": null
                 }
-            ]
+            ],
+            "vulnerabilities": []
         }
 
     :statuscode 200: no error
 
+    The ``vulnerabilities`` array is always empty for this endpoint. To get a list
+    of known vulnerabilities for a version of a project, refer to the endpoint
+    documented below.
```

Comment:
This endpoint defaults to the latest release -- if the latest release has a vulnerability, this would not be empty.

Reply:
Fixed.